### PR TITLE
June hco pr

### PR DESCRIPTION
**Pull request by DirectedSoul1**
**What this PR does / why we need it**:
New fixes for HCO minikube Blog

Just to keep the fork clean

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by DirectedSoul1** [comment]
@iranzo  please help review this new PR, Thanks

File: _posts/HCO-minikube.markdown [hunk 1]
```diff
@@ -0,0 +1,149 @@
+**HCO on minikube**
```
Comment (iranzo) [inline _posts/HCO-minikube.markdown]: Filename not conforming to date-title.markdown

Comment (DirectedSoul1) [reply _posts/HCO-minikube.markdown]: Changed the filename. as suggested

**Comment by DirectedSoul1** [comment]
> Hi,
> Remove the screenshoot* named files
> 
> And the one for HCO without date in the filename
Deleted the HCO file without the date
> 
> Thanks!
> Pablo

**Comment by iranzo** [comment]
Thanks for deleting the files!

**Pull request merged**